src/system/RenderableSystem.lua

```diff
@@ -20,6 +20,27 @@ function RenderableSystem:setScene ( tag )
 
 end
 
+function RenderableSystem:update( dt )
```

Comment:
I notice that our resting FPS is already down to 46 FPS on my computer. It was 60 FPS before, but I was aware that when the map was panned, it'd dive to 30FPS (and now it dips to around 23).

I think the culprit here is the as-fast-as-possible traversal of the scenegraph on each and every frame. I think I will make an issue for a Pub-Sub module in the game, which can broadcast '0.1 seconds have gone by' to Renderable/Animatable components, reducing how heavily this updateHeirarchy hits our computers.

Given this, we could perhaps combine Renderable/Animatable components into the same thing: Renderables simply being an Animatable with a single frame that doesn't subscribe to timeupdates? 

Might be worth a brief talk on wechat tomorrow on Friday: some useful choices to consider.